backend/functions/lib/testing/testing-utilities.ts

```diff
@@ -1,11 +1,34 @@
 import { ApplicationContext } from '../adapters/types/basic';
 import { applicationContextCreator } from '../adapters/utils/application-context-creator';
+import { HttpRequest } from '@azure/functions';
+import { MockData } from '../../../../common/src/cams/test-utilities/mock-data';
 const functionContext = require('azure-function-context-mock');
 
 // TODO: Need to refactor each test that needs an application context to use this convenience function.
```

Comment:
We should probably just remove this. I think we've gotten them all by now, but personally I don't fancy scouring the codebase for tests that should use this. I suppose we could search for instances of line 5 and double check those, but anything more involved than that sounds pointless.

Reply:
This TODO is addressed now I believe...